tests/integration/high_availability/high_availability_helpers.py

```diff
@@ -74,15 +77,21 @@ async def get_application_name(ops_test: OpsTest, application_name: str) -> str:
     return None
 
 
-async def ensure_n_online_mysql_members(ops_test: OpsTest, number_online_members: int) -> bool:
+async def ensure_n_online_mysql_members(
+    ops_test: OpsTest, number_online_members: int, mysql_units: Optional[List[Unit]] = None
+) -> bool:
```

Comment:
Might be nice to have a verification that the N members online are the N members running in Juju (verification that these two are in sync.) perhaps via IP address?

Reply:
i believe that this check would not be as productive: all units are in `active` state even though one of units has its mysqld paused due to a `sigstop`. i also believe that checking that the remaining members are the only online members and part of the same cluster would be sufficient

Reply:
Sure sounds good!